geonode/harvesting/harvesters/geonode.py

```diff
@@ -144,6 +161,20 @@ def get_extra_config_schema(cls) -> typing.Dict:
                 "resource_title_filter": {
                     "type": "string",
                 },
+                "start_date_filter": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "end_date_filter": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "keywords_filter": {
+                    "type": "array",
+                },
+                "categories_filter": {
+                    "type": "array",
```

Comment:
Please add a section describing the items of the array

```suggestion
                "keywords_filter": {
                    "type": "array",
                    "items": {
                        "type": "string",
                    }
                },
                "categories_filter": {
                    "type": "array",
                    "items": {
                        "type": "string",
                    }
```

Reply:
updated @ricardogsilva